Library/Formula/imply.rb

```diff
@@ -0,0 +1,27 @@
+class Imply < Formula
+  desc "Exploratory Analytics Powered By Druid"
+  homepage "http://imply.io/"
```

Comment:
This doesn't seem to point to the homepage for this actual software. Can you point to the GitHub repository?